### ISSUE TITLE
OrangeStorm Giga profile

### ISSUE THREAD

**Issue by RadovanPelka** [SoftFever/OrcaSlicer#6256]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Hello, could we add a new printer profile for OrangeStorm Giga? Link here: https://www.elegoo.com/products/orangestorm-giga

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

![CleanShot 2024-07-26 at 16 13 26@2x](https://github.com/user-attachments/assets/efaa9647-a29a-46e7-b9ad-8deefd702532)

**Comment by tlhintoq** [comment]
An 'official' one with the slicer would be nice.
There's this one as a starting point if it helps.
https://www.printables.com/model/926881-elegoo-orange-storm-giga-orca-slicer-profile-v1

**Comment by algocrypto** [comment]
I want to upvote this request to add Official version for Elegoo Giga Orangestorm printer profile in Orca Slicer.

**Comment by tlhintoq** [comment]
@Noisyfox 
I would urge you to pull in this guy's profile and work.  
Its terrific.  Its a profile many of us Giga owners are already adopting so it would REALLY cut down on foreseable problems where we're all using this already then Orca comes out with something different: That would be a mess.

https://github.com/TNTBA/OrangeStorm-Giga-Profile-for-Orca

Includes all the right json files, bed stl models and textures to look right, printer installation dialog selection.
Basically the guy has done all the work needed to add a proper machine profile - you just need to pull it in.

Plus he's incorprating fluidd config files for smart heatbed selection.  IE: If the model is only on bed 1 then only bed 1 turns on.
![image](https://github.com/user-attachments/assets/61cdfeba-7092-4ae3-890b-abd055f1578a)

![image](https://github.com/user-attachments/assets/18456b79-42c4-45db-96aa-14a59e596e40)

**Comment by tlhintoq** [comment]
Elegoo has released what they are calling either Elegoo slicer or giga slicer depending on where you read that.
They have included official profile Elegoo files for Orca and a decently cleaned up subdirectory hierarchy.
While it would have been nice of Elegoo just acted as a contributor here and participated in a true open-source mentality, instead they decided to fork Orca and brand it.  Not thrilled but no point crying about it.

Since Orca still hasn't created its own profile, how about reverse-merging their work to Orca slicer?  That keeps all the profiles for all the Elegoo machines in-sync so all of us users aren't fighting between two slicers or having 10,000 users all doing their own reverse merging every time one slicer or the other does an update.

Even @TNTBA stated that the official profiles are better than his wonderful work that rolled out before anyone elses.
![image](https://github.com/user-attachments/assets/d5ee7e0e-d783-4edb-b3ff-e3b2454ca039)
[giga profiles.zip](https://github.com/user-attachments/files/17068020/giga.profiles.zip)

**Comment by cochcoder** [comment]
> I am new to coding in general but with a lot of help from Chat GPT, I have created something that is working really well. I hope anyone who wishes to try this out gives it a shot! I am hoping to work with Orca Slicer Devs in the future to get this fully incorporated with the Orca Slicer project. The best way you can contribute is by tuning your print profiles so we get prints optimized for this machine. Thank you so much @tlhintoq for your help with this! :)
> 
> 
> 
> https://github.com/TNTBA/OrangeStorm-Giga-Profile-for-Orca

@TNTBA Would you say these profiles are better than the ones provided by Elegoo? I'm in the works of adding this printer to Orca Slicer and want to make sure that I'm adding the best profiles possible. 

Has anyone else tested these?

**Comment by tlhintoq** [comment]
[Elegoo.zip](https://github.com/user-attachments/files/19041796/Elegoo.zip)

I *think*  I merged the latest machine profiles and cleaned up the hierarchy to include the Elegoo Centauri and Carbon.
Anyone want to merge them as their own work and double check what I did while you're at it?  
CC:  @Noisyfox 
`\resources\profiles\Elegoo\machine`
